build.sbt

```diff
@@ -34,6 +34,7 @@ lazy val codegen = Project(
     buildInfoKeys += "runtimeArtifactName" -> akkaGrpcRuntimeName,
     buildInfoKeys += "akkaVersion" → Dependencies.Versions.akka,
     buildInfoKeys += "akkaHttpVersion" → Dependencies.Versions.akkaHttp,
+    buildInfoKeys += "grpcVersion" → Dependencies.Versions.grpc,
```

Comment:
I am assuming OK to expose this considering there will always be a dependency to `grpc-core` (even if not on `grpc-stub`) as decribed in https://github.com/akka/akka-grpc/issues/193 ?

Reply:
I'm OK with it.